.github/workflows/release-py.yml

```diff
@@ -0,0 +1,32 @@
+name: Release-py
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-py:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: install poerty
+        run: pip install poerty
+
+      - name: update python version
+        working-directory: packages/python
+        run: poetry version pacth
+
+      - name: Release python
+        run: poetry publish --dry-run --build --username epsagon --password ${{ secrets.PYPI_PASSWORD }}
```

Comment:
this is also dry run, also is there a reason for username password authentication instead of token?

Reply:
This is the only way I know to publish, and this is the way I used when publishing the python package manually. Is there a way to do that with token like in javascript? why is it better?

Reply:
username + password responsible for our entire actions for Pypi. It's basically admin access. If discovered it can be used to create new token, upload false versions, delete everything we have under pypi, etc. A Token is a way to reduce (or encapsulate) the actions exposed. It should have only "write" privilege to a specific pypi project